frontend/.storybook/main.ts

```diff
@@ -1,7 +1,7 @@
 import type { StorybookConfig } from "@storybook-vue/nuxt"
 
 const config: StorybookConfig = {
-  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
+  stories: ["../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
```

Comment:
We don't have any `.mdx` files, so Storybook was always printing warnings about this pattern.